src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -269,7 +269,8 @@ export class ViolationTicketService {
         ticketNumber: this.ticket.ticket_number,
         time: this.datePipe.transform(this.ticket.issued_date, "HH:mm")
       };
-      if (this.dateDiff(this.ticket.issued_date) <= 30) {
+      const ticketNumber = this.ticket.ticket_number.charAt(0);
+      if ((this.dateDiff(this.ticket.issued_date) <= 30 && ticketNumber == 'E') || (this.dateDiff(this.ticket.issued_date) <= 45 && ticketNumber == 'S')) {
```

Comment:
You can use the ticketType directly instead of creating a new const

Reply:
ok thanks